src/view/com/post-thread/PostThreadItem.tsx

```diff
@@ -645,9 +648,11 @@ function ExpandedPostDetails({
       <Text style={pal.textLight}>{niceDate(post.indexedAt)}</Text>
       {needsTranslation && (
         <>
-          <Text style={pal.textLight}> • </Text>
+          <Text style={pal.textLight}>•</Text>
```

Comment:
Lost padding space -- need to either preserve it or replace with styles

Reply:
Done